reference/src/glossary.md

```diff
@@ -1,5 +1,57 @@
 ## Glossary
 
+#### Aliasing
+
+(Please note: a full aliasing model for Rust has not yet been constructed, but
+at the moment we can give the following definition. The most developed potential
+aliasing model so far is known as "Stacked Borrows", and can be found
+[here](https://github.com/Lokathor/unsafe-code-guidelines/blob/lokathor/wip/stacked-borrows.md).)
```

Comment:
Please don't link to your fork. Link to the official repository's master branch.

Reply:
oh whoops. didn't mean that one